src/components/Nav/Nav.js

```diff
@@ -0,0 +1,319 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import TopBanner from './TopBanner';
+import { flexSet } from '../../styles/Variable';
+
+function Nav(props) {
+  const location = useLocation();
+
+  return (
+    <>
+      <TopBanner />
+      <MainNavContainer>
+        <GoToMainPageLink to="/">은행의집</GoToMainPageLink>
+        <LeftMenuCategoryWrapper>
+          <LeftMenuCategorySelected>커뮤니티</LeftMenuCategorySelected>
+          <LeftMenuCategoryUnselected>스토어</LeftMenuCategoryUnselected>
+          <LeftMenuCategoryUnselected>인테리어시공</LeftMenuCategoryUnselected>
+        </LeftMenuCategoryWrapper>
+        <RightMenuCategoryWrapper>
+          <SearchBoxWrapper>
+            <FaSearchIcon className="fas fa-search" />
+            <SearchBoxInput
+              aria-label="커뮤니티, 스토어 및 인테리어시공 검색"
+              placeholder="은행의집 통합검색"
+              type="search"
+            />
+            <SearchBoxButton>
+              <FaCameraIcon className="fas fa-camera" />
+            </SearchBoxButton>
+          </SearchBoxWrapper>
+          <CartIconWrapper>
+            <FaShoppingCartIcon className="fas fa-shopping-cart" />
+          </CartIconWrapper>
+          <LoginAndSignUpWrapper>
+            <GoToLoginPageLink to="/login">로그인</GoToLoginPageLink>
+            <GoToSignUpPageLink to="/signup">회원가입</GoToSignUpPageLink>
+          </LoginAndSignUpWrapper>
+          <WritingMenuWrapper>
+            <WritingMenuButton>
+              <GoToWritingPageLink to="/writing">글쓰기</GoToWritingPageLink>
+              <FaChevronDownIcon className="fas fa-chevron-down" />
+            </WritingMenuButton>
+          </WritingMenuWrapper>
+        </RightMenuCategoryWrapper>
+      </MainNavContainer>
+      <BottomNavContainer>
+        <CommunityMenuWrapper>
+          {COMMUNITY_CATEGORY_LIST.map(communityCategory => {
+            return (
+              <CommunityMenuCategories
+                key={communityCategory.menuName}
+                to={communityCategory.path}
+                isSelected={location.pathname === communityCategory.path}
+              >
+                {communityCategory.menuName}
+              </CommunityMenuCategories>
+            );
+          })}
```

Comment:
너무 깔끔하게 잘 해주셨네요!
한가지 문법적으로 더 적용해볼 수 있는 부분은 communityCategory 자체가 객체이기때문에 인자로 받는 자리에서 바로 구조분해할당 해줄 수 있습니다.
기능상의 차이는 하나도 없고 가독성적인 견해도 뭐가 더 좋다라고 말할 수는 없고 회사마다의 컨벤션에 따라야되는거지만
이렇게 할 수 있다로 알아두시기만 해도 좋을 것 같아요!
```suggestion
          {COMMUNITY_CATEGORY_LIST.map(({menuName, path}) => {
            return (
              <CommunityMenuCategories
                key={menuName}
                to={path}
                isSelected={location.pathname === communityCategory.path}
              >
                {menuName}
              </CommunityMenuCategories>
            );
          })}
```